homeassistant/components/remote/__init__.py

```diff
@@ -38,8 +41,12 @@
 
 SCAN_INTERVAL = timedelta(seconds=30)
 SERVICE_SEND_COMMAND = 'send_command'
+SERVICE_SEND_COMMANDS = 'send_commands'
```

Comment:
We should not add a new service. Instead, by using `cv.ensure_list` you can make the existing `send_command` support multiple commands and stay backwards compatible.